manubot/cite/tests/test_url.py

```diff
@@ -38,6 +38,8 @@ def test_get_url_csl_item_zotero_manubot():
     assert [int(x) for x in csl_item['issued']['date-parts'][0]] == [2018, 12, 18]
 
 
+# Note: this test had a temporary failure, maybe due to performance of 
+#       transaltion server.
```

Comment:
Put this in the docstring so it shows up when the test fails. Typo in `transaltion`.

FYI this is our most common build failure. It seems that sometimes translation-server returns a different title for the same URL. A real mystery. Probably should inquire upstream at some point or change the test. Anyways, this note is helpful.

Reply:
Actually what is we just modified the test to accept a title of `pandas-dev/pandas` as well as a title of "Flexible and powerful data analysis...". Then we could add a `TODO` comment to investigate further why tranlsation-server returns different titles intermittently while avoiding our tests from failing.